_includes/sidebar.html

```diff
@@ -12,15 +12,22 @@
     <nav class="sidebar-nav">
       {% for sections in site.contents[page.lang] %}
         {% assign _section = sections[0] | strip %}
+        {% if page.section == _section %}
+          {% assign opened = true %}
+        {% elsif page.section == "home" and forloop.index0 == 1 %}
+          {% assign opened = true %}
```

Comment:
I didn't run that code. So please make sure it works.
the proper solution would be to create a config variable called "first_section", but that would be overkill IMO.

I will run your changes tomorrow as I haven't had time to do it yet (all my suggestions have been made by reading code not by viewing the live version :-P), and report back.